python-sdk/nuimages/nuimages.py

```diff
@@ -659,29 +671,109 @@ def get_trajectory(self,
 
         return translations, key_index
 
+    def get_segmentation(self,
+                         sd_token_camera: str) -> Union[np.ndarray, np.ndarray]:
+        """
+        Produces two segmentation masks as numpy arrays of size H x W each, where H and W are the height and width
+        of the camera image respectively:
+            - semantic segmentation mask: A mask in which each pixel is an integer value between 0 to C (inclusive),
+                                          where C is the number of categories in nuImages. Each integer corresponds to
+                                          the index of the class in the category.json
+            - instance segmentation mask: A mask in which each pixel is an integer value between 0 to N, where N is the
+                                          number of objects in a given camera sample_data. Each integer corresponds to
+                                          the order in which the object was drawn into the mask.
```

Comment:
I would remove the word "segmentation" from "semantic segmentation mask" and "instance segmentation mask", as instance segmentation covers semantics and instances.